src/Libraries/RevitNodesUI/SelectionCombo.cs

```diff
@@ -0,0 +1,274 @@
+﻿using Autodesk.Revit.DB;
+using CoreNodeModels;
+using DSRevitNodesUI.Controls;
+using Dynamo.Controls;
+using Dynamo.Graph.Nodes;
+using Dynamo.Nodes;
+using Dynamo.Wpf;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CoreNodeModelsWpf.Nodes;
+using Autodesk.DesignScript.Runtime;
+using ProtoCore.AST.AssociativeAST;
+using Dynamo.Wpf.Nodes.Revit;
+using Microsoft.Practices.Prism.Commands;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using RevitServices.Persistence;
+using Autodesk.Revit.UI.Selection;
+using Dynamo.Logging;
+using DSCore;
+
+namespace Dynamo.ComboNodes
+{
+
+    #region NodeModels
+
+    [NodeName("Select Model Elements By Category"),
```

Comment:
Hi @SHKnudsen , I noticed this node use a new UI - ComboControl. When you save your Dynamo script, which contains node "Select Model Elements By Category". The dropdown list value can't be saved in the script. Then open the saved Dynamo script again, you will find that the dropdown list has no value, and the drop-down options are all empty.

Reply:
ah i see, working on fix @ZiyunShang

Reply:
@ZiyunShang should be fixed now, let me know if there are any issues.